gitlab.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "GitLab",
+  "release": "11.3-RELEASE",
+  "artifact": "https://github.com/pandorixtech/iocage-plugin-gitlab.git",
+  "pkgs": [
+    "www/gitlab-ce",
```

Comment:
Nothing crucial, but wouldn't it be more consistent to either use the category prefix for all package or for none? I.e. just `gitlab-ce` works fine here too.

Reply:
No problem! :) 
category prefix removed.
Thank you for your review.

Reply:
Cool, thanks for fast fix =)